entrypoint.sh

```diff
@@ -23,12 +26,9 @@ echo "Start: Sonar Scan"
 sh -c "/scripts/coverage_scan.sh"
 echo "End: Sonar Scan"
 
-echo "Container Push: $INPUT_CONTAINER_PUSH_ENABLED"
-if [ "$INPUT_CONTAINER_PUSH_ENABLED" = 'true' ]; then
-  echo "Start: Publish Image to ECR"
-  /scripts/publish.sh
-  echo "End: Publish Image to ECR"
-fi
+echo "Start: Publish Image to ECR"
+source actions-collection/collection.sh
```

Comment:
why are you running trivy.sh twice as publish.sh internally runs trivy.sh
docker_publish(){  echo "START: Publish.sh"  sh ./scripts/publish.sh  echo "END: Publish.sh"  echo "START: trivy_scan.sh"  sh ./scripts/trivy_scan.sh  echo "END: trivy_scan.sh"}

Reply:
My mistake. Added a PR for https://github.com/variant-inc/actions-collection/pull/1

Reply:
Corrected in this PR: https://github.com/variant-inc/actions-collection/pull/3

Could you take a look at it?